acm/odh-edge/apps/bike-rental-app/bike-rental-inference-generator-cronjob.yaml

```
@@ -0,0 +1,65 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: bike-rental-inference-generator
+spec:
+  schedule: "*/5 * * * *"  # This will run the job every hour
```

```suggestion
  schedule: "*/5 * * * *"
```
^This comment has already fallen out of sync

---

Done